src/test/java/teammates/test/cases/testdriver/TestngTest.java

```diff
@@ -0,0 +1,90 @@
+package teammates.test.cases.testdriver;
+
+import static org.testng.AssertJUnit.assertTrue;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.testng.annotations.Test;
+
+import teammates.common.util.FileHelper;
+import teammates.test.cases.BaseTestCase;
+
+public class TestngTest extends BaseTestCase {
+
+    List<String> tests = new ArrayList<String>();
+    String testngXmlAsString;
+
+    @Test
+    public void checkTestsInTestng() throws FileNotFoundException {
+        verifyTestngXmlExists();
+        readTestngAsString();
+        
+        getTestFiles();
+        excludeFilesNotInTestng();
+
+        verifyTestngContainsTests();
+    }
+    
+    private void verifyTestngXmlExists() {
```

Comment:
Instead of writing a method specifically to check this file, write a `FileHelper.isFileExists(String fileName):boolean` and use that.